resource-definitions/README.md

```diff
@@ -0,0 +1,127 @@
+# Secretless Broker Custom Resource Definitions (CRDs)
+
+This folder contains most of the resources for dealing with CRDs in the context of
+this codebase. CRDs are planned to be used for dynamically updating the broker
+configuration across the cluster when the API is changed.
+
+# Examples
+
+To manually create the CRD, let's fist ensure that we have none already defined:
+```
+$ kubectl get crd
+No resources found.
+```
+
+Now lets apply our CRD file template:
+```
+$ kubectl apply -f ./secretless.yaml
```

Comment:
can this be named to more clearly indicate what the file contains? maybe secretless-config-crd.yaml?

Reply:
or just secretless-resourcedefinition.yaml, following the example [here](https://kubernetes.io/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions/#create-a-customresourcedefinition)

Reply:
also, in that example they call `kubectl create -f resourcedefinition.yaml`, not `kubectl apply` - what's up with the difference?